README.md

```diff
@@ -0,0 +1,51 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Purchase
+
+> This is a budgeting app built with Ruby on Rails.
+
+## Built With
+
+- Ruby on Rails
+- Linters
+
+## Install
+
+- Install Ruby based on your OS from [HERE](https://www.ruby-lang.org/en/downloads/)
+
+## Setup
+
+1. Clone the project using git-bash or Githup Desktop. 
+~~~bash
+git clone https://github.com/vechicin/Purchase.git
+~~~
+OR
+~~~bash
+git clone git@github.com:vechicin/Purchase.git
+~~~
+2. Open the project folder with VSCode or any Editor.
```

Comment:
- [ ]  kindly I would suggest adding some screenshots of your project in your `README`.